PowerEditor/src/WinControls/DocumentMap/documentMap.cpp

```diff
@@ -442,14 +444,36 @@ INT_PTR CALLBACK DocumentMap::run_dlgProc(UINT message, WPARAM wParam, LPARAM lP
 	return DockingDlgInterface::run_dlgProc(message, wParam, lParam);
 }
 
+COLORREF ViewZoneDlg::_focus = RGB(0xFF, 0x80, 0x00);
+COLORREF ViewZoneDlg::_frost = RGB(0xFF, 0xFF, 0xFF);
+
+void ViewZoneDlg::setColour(COLORREF colour2Set, ViewZoneColorIndex i)
+{
+	switch (i)
+	{
+		case ViewZoneColorIndex::focus:
+		{
+			_focus = colour2Set;
+			break;
+		}
+
+		case ViewZoneColorIndex::frost:
+		{
+			_frost = colour2Set;
+			break;
+		}
+
+		default:
+			return;
+	}
+}
+
 void ViewZoneDlg::drawPreviewZone(DRAWITEMSTRUCT *pdis)
 {
 	RECT rc = pdis->rcItem;
-	
-	const COLORREF orange = RGB(0xFF, 0x80, 0x00);
-	const COLORREF white = RGB(0xFF, 0xFF, 0xFF);

```

Comment:
@donho 

> IMO, the background colour should remain as the main edit zone - think of Dark Mode. Don't you think so?

Part outside focus is using by default white color (`RGB(0xFF, 0xFF, 0xFF)`) so it may seems like there is no change in background color.

Those images are examples, and background color is changed to make notice of difference,